cypress/integration/AddNewOpeningPeriod.spec.ts

```diff
@@ -52,7 +52,12 @@ describe('User adds a new opening period', () => {
     // Try submit form
     cy.get('[data-test=publish-opening-period-button]').click();
 
-    // On successful creation a success notification should appear. Let's test that this is the case
+    // On successful creation a redirect to resource page and a success notification should appear. Let's test that this is the case
```

Comment:
Would it be more clear like this: On successful creation user is redirected to resource page and a success notification should appear?

Reply:
Yes, I'll fix